inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -330,6 +354,22 @@ void LayerTestsCommon::ConfigureNetwork() {
             out.second->setPrecision(outPrc);
         }
     }
+
+    // Reshape CNNNetwork before load to the plugin in dynamic scenario
+    if (!inputDynamicShapes.empty()) {
+        auto params = function->get_parameters();
+        std::map<std::string, ngraph::PartialShape> inputDataMap;
+        ASSERT_EQ(params.size(), inputDynamicShapes.size());
+        for (size_t i = 0; i < inputDynamicShapes.size(); i++) {
+            ngraph::PartialShape dynamicShape = inputDynamicShapes[i];
+            if (dynamicShape.rank() == 0) {
+                continue;
+            }
+            std::string inputName = params[i]->get_friendly_name();
+            inputDataMap.insert({inputName, dynamicShape});
+        }
+        cnnNetwork.reshape(inputDataMap);
```

Comment:
We should not use this API since it will not be released for 2022.1.
For dynamic shapes we need to use new OpenVINO 2.0 API

Reply:
Secondly, I don't understand at all - why do we need reshape?
I think the idea of dynamic shapes support - is to compile network with dynamic shapes w/o any reshape to static shapes

Reply:
discussed offline